### PR TITLE
Add Google Analytics tracking for 2SV actions

### PR DESCRIPTION
**Pull request by boffbowsh**
This spreadsheet contains all the proposed categories, actions and labels, and will need reviewing too:
https://docs.google.com/a/digital.cabinet-office.gov.uk/spreadsheets/d/1FZk_x1GFuktD-FpglP7FZq4bo_HBcsKbySRMXNcgfLA/edit?usp=sharing

Todo:

- [x] Sign off on proposed tracking labels etc from Paul Cronk
- [x] Merge and update govuk_admin_template

### CODE REVIEW

File: app/views/devise/two_step_verification/show.html.erb [hunk 1]
```diff
@@ -92,14 +92,14 @@
                     placeholder: 'Enter 6-digit code',
                     autocomplete: 'off' %>
                 </div>
-                <button name="submit" id="submit_code" class="btn btn-lg btn-success">
+                <button name="submit" id="submit_code" class="btn btn-lg btn-success js-track" data-track-label="finish">
```
Comment (fofr) [inline app/views/devise/two_step_verification/show.html.erb]: If we don't use a custom label we'll capture both scenarios. Is that preferable?

**Comment by boffbowsh** [comment]
Updated and rebased, addressing comments from spreadsheet too

File: app/views/devise/two_step_verification/show.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <% content_for :title, title %>
 <% invalid_code_entered = flash[:invalid_code] %>
 
-<h1><%= title %></h1>
+<h1 data-module="auto-track-event" data-track-action="title" data-track-label="<%= title%>"><%= title %></h1>
```
Comment (fofr) [inline app/views/devise/two_step_verification/show.html.erb]: Why are we tracking this as an event?

Comment (boffbowsh) [inline app/views/devise/two_step_verification/show.html.erb]: Is there an easier way of differentiating the pages?

Comment (fofr) [inline app/views/devise/two_step_verification/show.html.erb]: A pageview is tracked with the page titled automatically.

Comment (boffbowsh) [inline app/views/devise/two_step_verification/show.html.erb]: Ah-ha, will remove in that case.

**Comment by fofr** [comment]
Looks good to merge once tests pass :+1:

**Pull request merged**